server/views/pages/prison-register/prisonDetails.njk

```diff
@@ -20,6 +20,30 @@
   </p>
 {% endmacro %}
 
+{% macro line(text) %}
+  {% if text and text.trim() != '' %}
+    <p> {{ text }} </p>
+  {% endif %}
+{% endmacro %}
+
+{% macro renderAddress(line1, line2, town, county, postcode, country) %}
+  {{ line(line1) }}
+  {{ line(line2) }}
+  {{ line(town) }}
+  {{ line(county) }}
+  {{ line(postcode) }}
+  {{ line(country) }}
+{% endmacro %}
+
+{% macro addressTitle(line1, line2, town) %}
+  {% if line1 and line1.trim() != '' %} {{ line1 }}
```

Comment:
Because we can have multiple addresses for a single prison, we thought it necessary to identify each address. As they don't have an "address name" I use the first line of the address that is not empty.  line 1 and line 2 of the address are optional but town is mandatory

Reply:
yes - I missed that one - will add